windows/build.bat

```diff
@@ -1,8 +1,11 @@
+echo "Parameter 1: version x.x.x"
+echo "Parameter 2: .pfx password"
+
 if "%1"=="" (
   echo "please provide a version x.x.x"
 ) else (
-  "C:\Program Files\Microsoft SDKs\Windows\v7.1\Bin\signtool.exe" sign /f "E:\certificates\troupe-cert.pfx" "E:\opt\Gitter\win32\Gitter.exe"
-  "C:\Program Files\Inno Setup 5\ISCC.exe" "E:\windows\gitter.iss"
-  "C:\Program Files\Microsoft SDKs\Windows\v7.1\Bin\signtool.exe" sign /f "E:\certificates\troupe-cert.pfx" "E:\artefacts\GitterSetup*"
-  rename "E:\artefacts\GitterSetup.exe" "GitterSetup-%1%.exe"
+  "C:\Program Files (x86)\Microsoft SDKs\Windows\v7.1A\Bin\signtool.exe" sign /f "%cd%\certificates\troupe-cert.pfx" /p "%2" "%cd%\opt\Gitter\win32\Gitter.exe"
+  "C:\Program Files (x86)\Inno Setup 5\ISCC.exe" "%cd%\windows\gitter.iss"
+  "C:\Program Files (x86)\Microsoft SDKs\Windows\v7.1A\Bin\signtool.exe" sign /f "%cd%\certificates\troupe-cert.pfx" /p "%2" "%cd%\artefacts\GitterSetup*"
+  rename "%cd%\artefacts\GitterSetup.exe" "GitterSetup-%1%.exe"
 )
```

Comment:
No and it can be removed, but I would prefer to keep until the next cycle in this sprint because I almost couldn't find a workaround for https://github.com/nodejs/node/issues/5160 and needed to use it.

It is being replaced by `./windows/build.js`